### ISSUE TITLE
Lost saved edits while editing in the UI

### ISSUE THREAD

**Issue by nickidlugash** [mapbox/mapbox-studio-classic#85]
I just lost about 2 hours of saved edits on a TM2 project while editing in the browser UI. I'm racking my brain, but I can't think of anything I did differently that immediately proceeded this. Perhaps I navigated away from localhost:3000 and then hit the back button (but I really can't recall if I did this). 

When saving, I had mostly used the save button, and occasionally hit command S.

@ian29 suggested I open a ticket about this. If anyone (@yhahn?)  wants to investigate this error, let me know if I can help in any way.

**Comment by yhahn** [comment]
@nickidlugash was it a new project? Currently if you never hit the **Save as** button on a new project it never gets written to the hard disk.

I'll look into fixing this and buying you a drink sometime : )

**Comment by nickidlugash** [comment]
@yhahn  good to know. Not sure if that was the case or not – here's what I did (which was probably terrible, terrible workflow): 

I copied a different project of mine from the tm2-custom-styles repo into another (non-repo) place on my computer. I renamed that folder to be my new project. Then I checked out a new branch of the repo, and dragged the new project folder into the repo. And then opened the project in TM2. I think I did hit the 'Save as' button, because I had to change the title of the map.

Any thoughts?
